tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot

```diff
@@ -302,11 +302,17 @@ Check data catalogue page contains superseded warning for archived publication (
     user checks page contains element    testid:superseded-by-link
 
 Check that superseded warning link takes user to superseding-publication data-catalogue step 1
+    [Documentation]    Failing due to https://dfedigital.atlassian.net/browse/EES-4269
```

Comment:
Is it possible to add a warning to the descriptions of these bug tickets to look for Robot Tests that mention their ticket ref, and to include them again as a part of the bug fixing please?

Reply:
Good Idea. Done.

Reply:
That's what the [Documentation] tag is for. For these types of issues, we typically comment on the JIra  ticket to remove the Failing tag when the bug has been fixed